addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -264,7 +272,7 @@ private String lookupCode(Command command, ChannelUID channelUID) {
                 code = convertHexToGC(code);
                 logger.debug("Converted hex code is: {}", code);
             } catch (HexCodeConversionException e) {
-                logger.warn("Failed to convert hex code to GC format");
+                logger.info("Failed to convert hex code to GC format");
```

Comment:
That's a weird message for an info. I would either expect it to be some bug and thus would want to see an error/warning including stacktrace or to only have that as debug.